src/decoration.cpp

```diff
@@ -149,23 +149,25 @@ void Decoration::resize_outline(Rectangle outline, const DecorationScheme& schem
 
     auto tile = inner;
     client_->applysizehints(&inner.width, &inner.height);
-    if (!false) { // formely: if (!tight_decoration)
-        // center the window in the outline tile
-        // but only if it's relative coordinates would not be too close to the
-        // upper left tile border
-        int threshold = settings_.pseudotile_center_threshold;
-        int dx = tile.width/2 - inner.width/2;
-        int dy = tile.height/2 - inner.height/2;
-        inner.x = tile.x + ((dx < threshold) ? 0 : dx);
-        inner.y = tile.y + ((dy < threshold) ? 0 : dy);
-    }
+
+    // center the window in the outline tile
+    // but only if it's relative coordinates would not be too close to the
+    // upper left tile border
+    int threshold = settings_.pseudotile_center_threshold;
+    int dx = tile.width/2 - inner.width/2;
+    int dy = tile.height/2 - inner.height/2;
+    inner.x = tile.x + ((dx < threshold) ? 0 : dx);
+    inner.y = tile.y + ((dy < threshold) ? 0 : dy);
 
     //if (RECTANGLE_EQUALS(client->last_size, rect)
```

Comment:
What is this about?

Reply:
The `// if (RECTANGLE_EQUALS…`?  I originally intended to save unnecessary redraws if the geometry does not change. (But with the "full" window decorations we have, it's not enough to check for the rectangle and the border width...).

Reply:
Ok. I just question commented out code when I see some flying by during reviews ;)